src/app/Dashboard/components/DashboardContainer.js

```diff
@@ -94,6 +109,69 @@ export class DashboardContainer extends Component {
     });
   };
 
+  handleClick = index => (event) => {
+    if (event.target.value === 'Select All') {
+      const filterBy = this.state.filterBy.map((item) => {
+        if (event.target.checked) {
+          return (item.checked = true), item;
+        }
+
+        return (item.checked = false), item;
+      });
+
+      this.setState({ filterBy, filteredUserActivities: null });
+    } else {
+      const filterBy = this.state.filterBy.slice();
```

Comment:
Why do we have this line of code, since we are not `slicing` with any `start` and `end` values?

Reply:
I want a copy. although a could you spread there. 🤔